psalm.xml

```diff
@@ -5,6 +5,8 @@
     xmlns="https://getpsalm.org/schema/config"
     xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
     errorBaseline="psalm-baseline.xml"
+    findUnusedBaselineEntry="false"
+    findUnusedCode="false"
 >
```

Comment:
Please make `findUnusedBaselineEntry` true. It is trivial to `psalm --update-baseline` to fix it.